spec/services/moves/reference_generator_spec.rb

```diff
@@ -10,8 +10,10 @@
   let(:existing_reference) { '12345678' }
   let!(:move) { create :move, reference: existing_reference }
 
+  # Faker uses random numbers, so this test will need to be changed if
```

Comment:
Faker can generate any custom of custom string, can't it?
If it can't we can just create our own method, it's just some random chars + some random number